src/kubeobjects/slice.go

```diff
@@ -11,6 +11,24 @@ func MountPathIsIn(volumeMounts []corev1.VolumeMount, mountPathToCheck string) b
 	return false
 }
 
+func MountPathIsReadOnly(volumeMounts []corev1.VolumeMount, mountPathToCheck string) bool {
```

Comment:
can you merge
`func MountPathIsReadOnly(volumeMounts []corev1.VolumeMount, mountPathToCheck string) bool` and `func MountPathIsReadWrite(volumeMounts []corev1.VolumeMount, mountPathToCheck string) bool` 
to single function:
`MountPathIs(volumeMounts []corev1.VolumeMount, mountPathToCheck string, readOnly bool) bool` ?